ops/ansible/roles/bfd-db-migrator/templates/migrator-monitor.sh.j2

```diff
@@ -65,18 +65,46 @@ function get_migrator_state {
 #######################################
 function migrator_is_running {
   local _state="$1"
-  local _code="$(jq -r .code <<< "$_state")"
-  local _status="$(jq -r .status <<< "$_state")"
+
+  local _code
+  local _status
+  _code="$(jq -r .code <<< "$_state")"
+  _status="$(jq -r .status <<< "$_state")"
   [ "$_code" = '(null)' ] && [ "$_status" = "0/0" ]
 }
 
+#######################################
+# Sets the migration schema version based on the monitor log
+# Globals:
+#   None
+# Arguments:
+#   None
```

Comment:
This function **does** take a positional argument `$1` for the current state of the migrator. Please document that.

Reply:
Good enough. Thanks! 😉